### DISCUSSION TITLE
What is the word on OpenPGP on Solo2?

### DISCUSSION THREAD

**Discussion by VA1DER** [solokeys/solo2#177] [Q&A]
The last [official answer](https://github.com/solokeys/solo2/discussions/105#discussioncomment-2418512) I have seen on OpenPGP is that when Nitrokey's OpenPGP app was working that it would be made to work on the Solo2.  It has been working on the Nitrokey 3 for a while now.  Is there any realistic likelihood that we'll see this available for the Solo2?  Is it possible to hack in support somehow now using the Nitrokey implementation?

**Comment by epragalakis** [comment]
bumping this,
its 2025, it would be good to get a definite answer :pray:
